inductiva/tasks/run_simulation.py

```diff
@@ -86,8 +86,9 @@ def run_simulation(
     logging.info(
         "Consider tracking the status of the task via CLI:"
         "\n\tinductiva tasks list --task-id %s", task_id)
-    logging.info("Or, tracking the logs of the task via CLI:"
-                 "\n\tinductiva logs %s", task_id)
+    logging.info(
+        "Or, tracking the logs of the task via CLI:"
+        "\n\tinductiva logs %s", task_id)
```

Comment:
why these changes? does yapf complain?
I don't like having all these line breaks

Reply:
Yapf was failing for this file (which I hadn't even changed). The changes are just the result of running yapf over that file and committing it

Reply:
Yapf is so annoying...